engineering/ember.md

```diff
@@ -74,6 +72,28 @@ fullName: computed('user.firstName', 'user.lastName', {
 })
 ```
 
+### Use read only computed property
+
+There are some computed properties is our code base are having getter and setter, allowing property to be changed. We're moving away from this practice. Computed property should be read only.
```

Comment:
@yininge Could we give some explanation / reason of why this is better? 🙏And not use `our code base`. This is a public rep so better to keep the context global